### PR TITLE
fix: notebook update

### PR DESCRIPTION
**Pull request by mbeach-aws**
*Issue #, if available:*
Simplify example notebook for Aria error mitigation.

*Description of changes:*
Added cost tracking to Aria example notebook. Refactored phase estimation circuit code to separate .py file.  

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: examples/braket_features/Error_Mitigation_on_Amazon_Braket.ipynb [hunk 1]
```diff
@@ -1,509 +1,433 @@
 {
```
Comment (speller26) [inline examples/braket_features/Error_Mitigation_on_Amazon_Braket.ipynb:1]: <p>Is the print necessary?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-braket-examples/pull/286/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>

Comment (virajvchaudhari) [inline examples/braket_features/Error_Mitigation_on_Amazon_Braket.ipynb:1]: <p>Should we be also adding the cost tracker information for this notebook, similar to <a href="https://github.com/aws/amazon-braket-examples/blob/main/examples/braket_features/Simulating_Noise_On_Amazon_Braket.ipynb" target="_blank">others</a>?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-braket-examples/pull/286/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

File: examples/braket_features/phase_estimation.py [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+    """Phase estimation circuit. The circuit has 1 query qubit and `n_precision_qubits`
+    precision qubits. The resolution of phase estimation is `(1/2**n_precision_qubits)*2*pi`.
+    The circuit has `n_precision_qubits+1` qubits. To make benchmark easier, the target phase
+    is integer multiple of the phase resolution, `(fraction/2**n_precision_qubits)*2*pi`.
+    For example, if `n_precision_qubits=3` and `fraction=3`, the target phase is `(3/8)*2*pi`.
+
+    Args:
+        n_precision_qubits (int): Number of precision qubits.
+        target_phase_in_resolution (int): Number of fraction, (1/2**n_precision_qubits)*2*pi.
+            The oracle phase is (fraction/2**n_precision_qubits)*2*pi.
```
Comment (virajvchaudhari) [inline examples/braket_features/phase_estimation.py:15]: Can we have the return statement in those doc-strings?

Comment (mbeach-aws) [reply examples/braket_features/phase_estimation.py]: updated

File: examples/braket_features/Error_Mitigation_on_Amazon_Braket.ipynb (hunk 1 above)
Comment (mbeach-aws) [reply examples/braket_features/Error_Mitigation_on_Amazon_Braket.ipynb]: <p>i'll leave the print in for now, so I can remember which task it is :P </p>

**Comment by ajberdy** [comment]
Maybe we can have a more descriptive PR/commit name?

**Comment by virajvchaudhari** [comment]
Can you confirm if we also need modifications for the the integ_tests based on these changes, or if we need to add anything for this notebook? I think we don't have tests for this.

**Review by speller26 — APPROVED** [review]

**Pull request merged**